.well-known/security.txt

```diff
@@ -0,0 +1,16 @@
+# If you would like to report or fix a bug
+# Create an issue or a pull request on the
+# Northstar Github page
+
+# Northstar Atlas (Master Server)
+https://github.com/R2Northstar/Atlas
+
+# Northstar Launcher
+https://github.com/R2Northstar/NorthstarLauncher
+
+# Northstar Game Scripts
+https://github.com/R2Northstar/NorthstarMods
+
+# Or notrify one of our developers on Discord
```

Comment:
`notrify`

Reply:
How did I miss this twice -_-

Updated